Components/AppTabNavigator/StocksTab.js

```diff
@@ -23,16 +23,76 @@ class StocksTab extends Component {
         companyNames: [],
         companySymbols: [],
       },
+      favoritedCompanies: {
+        companyNames: [],
+        companySymbols: [],
+      },
     };
   }
 
   componentDidMount() {
-    //NetInfo.isConnected.fetch().then(isConnected => {
     if (true) {
       this.getStockCardData();
+      this.readFavorites();
     }
+
+    this.willFocusSubscription = this.props.navigation.addListener(
+      'focus',
+      () => {
+        this.readFavorites();
+        console.log("BACK BUTTON");
+      }
+    );
+
+  }
+
+  componentWillUnmount() {
+    this.willFocusSubscription.remove();
```

Comment:
I got a TypeError saying .remove() didn't exist, I'm going to try adding this.willFocusSubscription = null to constructor to see if that fixes this